docs/content/preview/api/ycql/ddl_create_index.md

```diff
@@ -116,6 +129,7 @@ When an index is created on an existing table, YugabyteDB will automatically bac
 - The `CLUSTERING ORDER BY` property can be used to set the ordering for each clustering column individually (default is `ASC`).
 - The `TABLETS = <num>` property specifies the number of tablets to be used for the specified YCQL index. Setting this property overrides the value from the [`--yb_num_shards_per_tserver`](../../../reference/configuration/yb-tserver/#yb-num-shards-per-tserver) option. For an example, see [Create an index specifying the number of tablets](#create-an-index-specifying-the-number-of-tablets).
 - Use the `AND` operator to use multiple index properties.
+- When setting a ttl on the index using `default_time_to_live`, please ensure that the ttl value is the same as that of the table's ttl. If they are different, it would lead to the index and the table being out of sync and would lead to unexpected behavior.
```

Comment:
```suggestion
- When setting a TTL on the index using `default_time_to_live`, please ensure that the TTL value is the same as that of the table's TTL. If they are different, it would lead to the index and the table being out of sync and would lead to unexpected behavior.
```

We should also note that the TTL will not be transactional. Meaning, suppose you are reading 10K rows using an index, if the expiration is now, you may read ALIVE rows from the index, but when going to the table the rows would be expired.